pkgs/os-specific/linux/mmc-utils/default.nix

```diff
@@ -10,6 +10,8 @@ stdenv.mkDerivation rec {
     sha256 = "0hkdzc71pdnscbpdpgwljcchiyancarldjyd0w609sy18bky833x";
   };
 
+  makeFlags = "CC=${stdenv.cc.targetPrefix}cc";
```

Comment:
I don't believe this is needed, here's cross-building to aarch64-musl:

```console
$ nix build -f channel:nixos-unstable --arg crossSystem '(import <nixpkgs/lib>).systems.examples.aarch64-multiplatform-musl' mmc-utils
$ ls -l result
lrwxrwxrwx 1 will users 91 May  3 16:34 result -> /nix/store/g5izr5yd8sydw6dizmnj8y2mlvwx28a5-mmc-utils-2015-11-18-aarch64-unknown-linux-musl
```

(where unstable is based on master, and doesn't have this commit yet)

Reply:
The same here.